### PR TITLE
Updating dev-setup to explain how to clone development repos

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: source/developer/dev-setup-windows.rst [hunk 1]
```diff
@@ -14,12 +14,18 @@ Installing Developer Components on Windows
 3. Download and install Node.js from https://nodejs.org/.
 4. Download and install Go 1.8 from https://golang.org/dl/.
 5. Install Yarn. Go to https://yarnpkg.com/en/docs/install#windows-tab and follow the installation instructions.
-6. Fork Mattermost on GitHub.com from https://github.com/mattermost/platform, then:
+6. Fork Mattermost server and webapp on GitHub from https://github.com/mattermost/mattermost-server and https://github.com/mattermost/mattermost-server, then:
```
Comment (lindy65) [inline source/developer/dev-setup-windows.rst]: Shouldn't this read "Fork Mattermost server and webapp on GitHub from https://github.com/mattermost/mattermost-server and https://github.com/mattermost/mattermost-webapp, then:"

**Comment by jespino** [comment]
@lindy65 you are completely right. Now fixed. Thanks!

**Comment by lindy65** [comment]
Thank you @jespino :)

**Comment by hmhealey** [comment]
Looks good to me

**Pull request merged**